.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
+/.github
```

Comment:
- [ ] Please make sure that you add `.github/workflow` and `linters.yml` inside it to make linters Github actions work, I see that you add the `.github` directory to the `.gitignore` file, and that's what causing the problem so kindly remove `.github` from `.gitignore` file.
- [ ] Kindly add `node_modules` directory to your `.gitignore` file.